include/wabt/ir.h

```diff
@@ -429,7 +429,8 @@ enum class ExprType {
   Unreachable,
 
   First = AtomicLoad,
-  Last = Unreachable
+  Last = Unreachable,
+  OpCodeRaw = Last + 1,  // virtual type
```

Comment:
correct us if we're wrong, but you should be able to put it into the enum as normal, between `Nop` and `RefIsNull`. you'll need to update the matching array in uh `ir.cc` (and maybe other places) but the alternative would be to make the code reject `OpCodeRaw` everywhere else.

also, how about calling this `UndecodedOpcodes` (between `Unary` and `Unreachable`)?